pkg/statedb/cell.go

```diff
@@ -22,6 +22,7 @@ var Cell = cell.Module(
 	"In-memory database",
 
 	cell.Provide(New),
+	cell.Provide(newDumpHandler), // REST API Handler
```

Comment:
nit(in case you rebase): you can collapse the provide's into one which I find
easier to read:

```
cell.Provide(
  New,
  newDumpHandler, // The /statedb/dump REST API handler
)
```